DEVELOPER_GUIDE.md

```diff
@@ -234,17 +234,54 @@ BUILD SUCCESSFUL in 7s
 
 #### Regression Tests
 
-Jenkins workflow regression tests typically output a .txt file into [tests/jenkins/jobs](tests/jenkins/jobs). For example, [TestHello.groovy](tests/jenkins/TestHello.groovy) executes [Hello_Jenkinsfile](tests/jenkins/jobs/Hello_Jenkinsfile) and outputs [Hello_Jenkinsfile.txt](tests/jenkins/jobs/Hello_Jenkinsfile.txt). If the job execution changes, the regression test will fail. To update the recorded .txt file run `./gradlew test -info -Ppipeline.stack.write=true` or update its value in [gradle.properties](gradle.properties).
+Jenkins workflow regression tests typically output a .txt file into [tests/jenkins/jobs](tests/jenkins/jobs).
+For example, [TestHello.groovy](tests/jenkins/TestHello.groovy) executes [Hello_Jenkinsfile](tests/jenkins/jobs/Hello_Jenkinsfile)
+and outputs [Hello_Jenkinsfile.txt](tests/jenkins/jobs/Hello_Jenkinsfile.txt). If the job execution changes, the regression test will fail.
+
+- To update the recorded .txt file run `./gradlew test -info -Ppipeline.stack.write=true` or update its value in [gradle.properties](gradle.properties).
+
+- To run a specific test case, run `./gradlew test -info -tests=TestCaseClassName`
```

Comment:
We could introduce the functionalities of different params here. 
e.g. `-info` will give the results in details. 
`-tests=TestCaseName` can be applied to run test on specific test class. 
`-Ppipeline.stack.write=true` will give the write access to update the `.txt` file

Reply:
`./gradlew --help` should give info about all the option params